### DISCUSSION TITLE
Doing mass updates

### DISCUSSION THREAD

**Discussion by christophe-fms** [netsuitephp/netsuite-php#139] [Q&A]
Hi all,

I'm doing a mass update for an ItemFulfillment, my request looks like this :
foreach items :
- i create a new packagelist
-  i create a new itemFulfillmentItemList
- i creat a new ItemFulfillmentPackageList
and  ending with $addResponse = $service->update($request);
end foreach

The problem is that it is taking a very long time if i have many items (probably because addresponse is in the loop). Is there any way  to do just 1 update call with all the items to update ?
Best regards

**Comment by paulasokolowicz** [comment]
hi! could you solve that?

also i am having some issues with updating itemfulfillment. can you help me with that?
